regimes/it/examples/hotel.json

```diff
@@ -39,7 +39,7 @@
     "name": "Mela S.r.l.",
     "inboxes": [
       {
-        "key": "codice-destinatario",
+        "key": "it-sdi-code",
```

Comment:
Mentioning just in case: Is it safe to assume this won't break any existing implementation (e.g. Amenitiz) and that we don't need a migration?

Reply:
So far, no one has used this in production, so I think we're good. Previous support was also a bit hacky without a specific constant to use. It'll be easy to add to the gobl.fatturapa tool though if anyone needs it.